deployment/refinery-modules/refinery/manifests/docker_.pp

```diff
@@ -0,0 +1,5 @@
+class refinery::docker_ {
```

Comment:
Because there is already a class`docker` and Puppet complains about a naming conflict